src/Components/UserModal/UserModal.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import { useState } from 'react';
+import './UserModal.scss';
+
+function UserModal({ data, userInfoClose, modalChangeHandle, backHandle }) {
+  const { title, button } = data;
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [emailValue, setEmailValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [isCheckingBox, setIsCheckingBox] = useState('');
+  const emailCheck = emailValue.includes('@');
+  const passwordCheck = passwordValue.length >= 5;
+  const isChecked = isCheckingBox === true;
+
+  function firstNameHandle(e) {
+    setFirstName(e.target.value);
+  }
+
+  function lastNameHandle(e) {
+    setLastName(e.target.value);
+  }
+
+  function emailHandle(e) {
+    setEmailValue(e.target.value);
+  }
+
+  function passwordHandle(e) {
+    setPasswordValue(e.target.value);
+  }
+
+  function checkBoxHandle(e) {
+    setIsCheckingBox(e.target.checked);
+  }
+
+  function isPossible(e) {
+    e.preventDefault();
+    if (title === '로그인') {
+      if (!(emailCheck && passwordCheck && isChecked)) {
+        alert('로그인 실패');
+      } else if (emailCheck && passwordCheck && isChecked) {
+        fetch('http://10.58.52.78:3000/users/signin', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json;charset=utf-8',
+          },
+          body: JSON.stringify({
+            email: emailValue,
+            password: passwordValue,
+          }),
+        })
+          .then(response => {
+            response.json();
+          })
+          .then(result => {
+            console.log(result);
+          });
+      }
+    } else if (title === '회원가입') {
+      if (!(emailCheck && passwordCheck && isChecked)) {
+        alert('회원가입 실패');
+      } else if (emailCheck && passwordCheck && isChecked) {
+        fetch('http://10.58.52.78:3000/users/signup', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json;charset=utf-8',
+          },
+          body: JSON.stringify({
+            firstName: firstName,
+            LastName: lastName,
+            email: emailValue,
+            password: passwordValue,
+          }),
```

Comment:
```suggestion
          body: JSON.stringify({
            firstName,
            LastName: lastName,
            email: emailValue,
            password: passwordValue,
          }),
```
객체의 key, value가 동일할 때는 이렇게 단축속성명을 적용할 수도 있습니다!

Reply:
그렇다고 key, value를 꼭 맞출 필요는 없으니 참고해주세요!